adagucserverEC/CCairoPlotter.cpp

```diff
@@ -1252,6 +1252,8 @@ void CCairoPlotter::drawBarb(int x, int y, double direction, double strength, CC
   CT::string text;
   text.print("%d", strengthInKnots);
   if (drawText) {
-    this->drawStrokedText(x - cos(direction + M_PI) * 15 - 5, y + sin(direction + M_PI) * 12 + 5, 0, text.c_str(), 12, 1 * drawOutline, outlineColor, barbColor);
+    // If speed is really low, draw the text below the circle
+    double textDirection = strengthInKnotsRoundedToFive <= 2 ? -M_PI / 2.1 : direction;
+    this->drawStrokedText(x - cos(textDirection + M_PI) * 15 - 5, y + sin(textDirection + M_PI) * 12 + 5, 0, text.c_str(), 12, 1 * drawOutline, outlineColor, barbColor);
```

Comment:
Maybe this is something out of the scope of the PR but good for the future refactoring, but I see a lot of "magic numbers" and it is difficult to follow their meaning (for example, why 15? why 2.1).

Reply:
Yes, this is for text placement of the number so it visually appears at the right place.
Since it is captured in tests, and since we want to refactor I would leave it like this.